netclient/wireguard/common.go

```diff
@@ -194,10 +194,7 @@ func InitWireguard(node *models.Node, privkey string, peers []wgtypes.PeerConfig
 		}
 	}
 	logger.Log(1, "interface ready - netclient.. ENGAGE")
-	if syncconf { // should never be called really.
-		fmt.Println("why here")
-		err = SyncWGQuickConf(ifacename, confPath)
-	}
+
```

Comment:
if syncconf check is removed the corresponding func arg should be removed and all calls to InitWireguard updated.